lldp/lldp.yaml

```diff
@@ -221,3 +230,92 @@ components:
             User must specify a value if mode is not auto.
           type: string
           x-field-uid: 3
+
+    Lldp.OrgInfo:
+      description: >-
+        The organization specific information configured in the Organization Specific TLV. 
+      type: object
+      required: [information]
+      properties:
+        oui:
+          description: >-
+            The organizationally unique identifier field shall contain the organization's OUI as defined in Clause 9 of IEEE 
+            Std 802. It is a 24 bit number that uniquely identifies a vendor, manufacturer, or other organizations. 
+          type: integer
+          format: uint32
+          minimum: 1
+          maximum: 6777215
+          default: 1
+          x-field-uid: 1
+        subtype:
+          description: >-
+            The organizationally defined subtype field shall contain a unique subtype value assigned by the defining organization.
+          type: integer
+          format: uint32
+          minimum: 1
+          maximum: 127
+          default: 1
+          x-field-uid: 2
+        information:
+          description: >-
+            Contains the organizationally defined information. The actual format of the organizationally defined information string 
+            field is organizationally specific and can contain either binary or alpha-numeric information that is instance specific 
+            for the particular TLV type and subtype. Alpha-numeric information are encoded in UTF-8 (as specified in IETF RFC 3629). 
+            Or include one or more information fields with their associated field-type identifiers, designators similar to those in 
+            the Management Address TLV.
+          $ref: '#/components/schemas/Lldp.OrgInfoType'  
+          x-field-uid: 3
+
+    Lldp.OrgInfoType:
+      description: >-
+        Contains either the Alpha-numeric information encoded in UTF-8 (as specified in IETF RFC 3629) or include one or more information fields with 
+        their associated field-type identifiers designators, similar to those in the Management Address TLV.
+      type: object
+      properties:
+        choice:
+          description: >-
+            In info mode the organizationally defined information contain either binary or alpha-numeric information encoded in UTF-8 
+            (as specified in IETF RFC 3629). Subinfo mode will contain a array of sub tlvs.
+          type: string
+          default: info 
+          x-field-uid: 1
+          x-enum:
+            info:
+              x-field-uid: 1
+            subinfo:
+              x-field-uid: 2
+        info:
+          description: >-
+            The organizationally defined information encoded in UTF-8 (as specified in IETF RFC 3629).
+          type: string
```

Comment:
In choice description, it is mentioned this can be binary . If that is correct string as the data type might not work for that option.

Reply:
If binary and alpha-numeric are both valid , either we might always want this to be be hex or there can be two supported formats , one hex ( for binary ) and other as a regular string.

Reply:
updated